File: problemas/1000/1042.js
const entrada = require('fs').readFileSync('/dev/stdin', 'utf-8').split(' ');
const [a, b, c] = entrada.map(x => Number(x));
const numeros = Array.from([a, b, c]);

numeros.sort((a, b) => a - b);
console.log(`${numeros[0]}\n${numeros[1]}\n${numeros[2]}\n`);
console.log(`${a}\n${b}\n${c}`);

File: problemas/1000/1042.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = int.parse(entrada[0]);
  final b = int.parse(entrada[1]);
  final c = int.parse(entrada[2]);

  final numeros = [a, b, c];
  numeros.sort();

  stdout.writeln('${numeros[0]}\n${numeros[1]}\n${numeros[2]}\n');
  stdout.writeln('$a\n$b\n$c');
}